tests/Unit/Domain.LinnApps.Tests/PartSupplierServiceTests/WhenCreatingAPreferredSupplierChangeForAPartForTheFirstTime.cs

```diff
@@ -127,27 +125,7 @@ public void ShouldUpdatePartSupplierRecords()
         [Test]
         public void ShouldInsertAPartHistoryRecord()
         {
-            this.PartHistory.Received().Add(Arg.Is<PartHistoryEntry>(x =>
-                x.Seq == 1
-                && x.PartNumber == "PART"
-                && x.OldMaterialPrice == null
-                && x.OldLabourPrice == null
-                && x.NewMaterialPrice == 3m
-                && x.NewLabourPrice == 0m
-                && x.OldPreferredSupplierId == null
-                && x.NewPreferredSupplierId == 2
-                && x.OldBomType == "A"
-                && x.NewBomType == "A"
-                && x.ChangedBy == 33087
-                && x.ChangeType == "PREFSUP"
-                && x.Remarks == "REMARKS"
-                && x.PriceChangeReason == "CHG"
-                && x.OldCurrency == null
-                && x.NewCurrency == "USD"
-                && x.OldCurrencyUnitPrice == null
-                && x.NewCurrencyUnitPrice == 100m
-                && x.OldBaseUnitPrice == null
-                && x.NewBaseUnitPrice == 3m));
+            this.PartHistory.Received().AddPartHistory(Arg.Any<Part>(), Arg.Any<Part>(), "PREFSUP", Arg.Any<int>(),"REMARKS","CHG");
```

Comment:
seems like this test is losing some specificity compared to the one its replacing. Not sure whether that really matters or not

Reply:
If I was worried about AddPartHistory I could probably write unit tests for that